code/game/objects/items/weapons/grenades/grenade.dm

```diff
@@ -19,6 +19,10 @@
 	prime()
 	..()
 
+/obj/item/weapon/grenade/acid_melt()
+	prime()

```

Comment:
Should provide for some interesting ghetto cluster grenades where you trigger an acid grenade on a pile of other grenades!